concent_api/core/payments/storage.py

```diff
@@ -86,7 +84,6 @@ def get_all_tx(self) -> List[Transaction]:
             for ethereum_transaction in PendingEthereumTransaction.objects.all()
         ]
 
-    @non_nesting_atomic(using='control')
```

Comment:
What was wrong here? You removed the decorator from the function but everything is still wrapped in `@atomic` so this should not change behavior in any way.

Reply:
Not everything, `self._get_nonce()` is not ;)